src/pages/index.tsx

```diff
@@ -241,15 +247,21 @@ function Home() {
             </Button>
           ) : (
             <form onSubmit={handleSend} style={{ display: 'flex' }}>
-              <Input
-                m={2}
-                px={2}
-                flex={1}
-                focusBorderColor="flame.600"
-                placeholder="Say something"
-                value={messageText}
-                onChange={(ev) => setMessageText(ev.target.value)}
-              />
+              <InputGroup>
+                <Input
+                  m={2}
+                  px={2}
+                  flex={1}
+                  focusBorderColor="flame.600"
+                  placeholder="Say something"
+                  value={messageText}
+                  maxLength={maximumMessageSize}
+                  onChange={(ev) => setMessageText(ev.target.value)}
+                />
+                <InputRightElement flexShrink={0} width={50} height={50}>
```

Comment:
We should make the limit text unselectable with:

```suggestion
                <InputRightElement flexShrink={0} width={50} height={50} pointerEvents="none">
```